plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -180,34 +180,34 @@ public void widgetSelected(SelectionEvent event) {
     });
   }
 
-  /** Create UI for specifying desired Maven Coordinates */
-  private void createMavenCoordinatesArea(Composite container, ModifyListener pageValidator) {
-    Group mavenCoordinatesGroup = new Group(container, SWT.NONE);
-    mavenCoordinatesGroup.setText(Messages.getString("MAVEN_PROJECT_COORDINATES")); //$NON-NLS-1$
-    GridDataFactory.defaultsFor(mavenCoordinatesGroup).span(2, 1).applyTo(mavenCoordinatesGroup);
-    GridLayoutFactory.swtDefaults().numColumns(2).applyTo(mavenCoordinatesGroup);
+  /** Create UI for specifying desired Maven coordinate */
```

Comment:
Standard usage in the Maven docs is plural. A groupId would be a singular coordinate but a groups ID, artifact ID, and version are three coordinates.